src/main/webapp/src/app/recommendation-form/recommendation-form.component.html

```diff
@@ -0,0 +1,74 @@
+<h1>Hidden Gems Recommendations</h1>
+<h4>Get a restaurant recommendation tailored just for you!</h4>
+
+<form [formGroup]="preferenceForm" (ngSubmit)="onSubmit()">
+  <h2>Your Restaurant Preferences</h2>
+  <p>Please select your restaurant preferences in the form below. If you have no preference for a field, please feel free to leave it blank</p>
+  
+  <h3>
+    <label for="cuisine">
```

Comment:
From what I understood, we cannot retrieve the cuisine of our hidden gems with our currently chosen APIs. In this case, you could keep this section and see if later on we can implement it (for example, we saw we can use the Zomato API for that, but have not actively planned to use it yet) and add a comment indicating that, or you could omit it for now and re-add it later if we decide to offer this option. It's up to you

Reply:
removed